src/popup/popup.js

```diff
@@ -0,0 +1,97 @@
+function main() {
+
+    const $loginbutton = document.getElementById('generate-button');
```

Comment:
I would rename the id to `login-button` to make its function more clear.

Reply:
This was renamed in 5d604c5f83ec276107c7331b4c57c03e12f1a4a9